src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -86,6 +86,8 @@ private void initFieldData() {
     keyGeneratorPane.setText("Key generator");
     keyGeneratorPane.setExpanded(true);
 
+    generatedKeyFld.setPromptText("Generating key...");
```

Comment:
I changed the font size for now to 20 and added a todo, it now says "generating" in the standard size